course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -101,7 +101,7 @@ private void createCourseBatch(Request actorMessage) throws Throwable {
               ResponseCode.invalidRequestParameter.getErrorMessage(), PARTICIPANTS));
     }
     CourseBatch courseBatch = JsonUtil.convert(request, CourseBatch.class);
-    courseBatch.setStatus(setCourseBatchStatus(actorMessage.getRequestContext(), (String) request.get(JsonKey.START_DATE)));
```

Comment:
"setCourseBatchStatus" - is this function still required? If not, delete that too.

Reply:
done